conda_build/noarch_python.py

```diff
@@ -24,7 +26,7 @@ def rewrite_script(fn):
     directory after it passes some sanity checks for noarch pacakges"""
 
     # Load and check the source file for not being a binary
-    src = join(config.build_prefix, 'bin', fn)
+    src = join(config.build_prefix, 'Scripts' if ISWIN else 'bin', fn)
```

Comment:
mmm, but in contrast to entry points, these exe's could not be translated to native linux/osx executables, so it could not be a noarch package. I think.